assets/airports/kokc.json

```diff
@@ -0,0 +1,3142 @@
+{
+    "airac": 2201,
+    "radio": {
+        "twr": "Rogers Tower",
+        "app": "Oke City Approach",
+        "dep": "Oke City Departure"
+    },
+    "icao": "KOKC",
+    "iata": "OKC",
+    "magnetic_north": 4.0,
+    "ctr_radius": 80,
+    "ctr_ceiling": 15000,
+    "initial_alt": 5000,
+    "position": [35.3930740, -97.6007617, "1296ft"],
+    "rangeRings": {
+        "enabled": true,
+        "center": [35.3930740, -97.6007617],
+        "radius_nm": 5.0
+    },
+    "has_terrain": true,
+    "wind": {
+        "angle": 330,
+        "speed": 4
+    },
+    "arrivalRunway": "36R",
+    "departureRunway": "36L",
+    "defaultMaps": ["Base Map"],
+    "airspace": [
```

Comment:
The Stillwater Shelf is only controlled by OKC above 5000ft, this should be reflected in the sim
![image](https://user-images.githubusercontent.com/13125311/152350530-388b06be-ee0c-48cb-9cb5-8654f855dc1e.png)